gems/rqrcode/2.2/rqrcode.rbs

```diff
@@ -0,0 +1,14 @@
+module RQRCode
+  class QRCode
+    type level = :l | :m | :q | :h
+    type mode = :number | :alphanumeric | :byte_8bit | :kanji
+    type segment = { data: String, mode: mode }
+
+    def initialize: (String) -> void
+    def as_svg: (?offset: Integer, ?fill: String | :white | :currentColor, ?color: String | :black | :currentColor, ?module_size: Integer, ?shape_rendering: "auto" | "optimizeSpeed" | "crispEdges" | "geometricPrecision", ?standalone: bool, ?use_path: bool, ?viewbox: bool, ?svg_attributes: Hash[untyped, untyped]) -> String
+    def as_png: (?fill: untyped, ?color: untyped, ?size: Integer, ?border_modules: Integer, ?module_px_size: Integer, ?border: Integer, ?resize_exactly_to: bool, ?resize_gte_to: bool) -> untyped
```

Comment:
Oops... I need to add types for chunky_png gem to avoid the CI errors. I'll do it tomorrow.